recipe/meta.yaml

```diff
@@ -61,12 +56,6 @@ about:
     Transfer data between system and GPU memory
     Report errors
```

Comment:
continue cleanup of the description field. I think the last two lines can be removed, or maybe there is a better description in intel's repo.

Reply:
This is the list of "Features" that intel provides. I can just remove all of this and leave the first line.